pkgs/development/python-modules/mlx/default.nix

```diff
@@ -46,13 +43,10 @@ let
       hash = "sha256-hbqV/2KYGJ1gyExZd5bgaxTdhl5+Gext+U/+1KAztMU=";
     };
 
-    patches = [
-      (replaceVars ./darwin-build-fixes.patch {
-        sdkVersion = apple-sdk_14.version;
-      })
-    ];
-
```

Comment:
I removed the newly added patch in favor of just using `xcbuild`.

I can revert this change, but I think it's better if we try to patch as few things as possible. (Even if we get two extra build-time deps)

Though

Reply:
Excellent.